components/compliance-service/ingest/ingestic/ingestic.go

```diff
@@ -20,6 +20,138 @@ type ESClient struct {
 	initialized bool
 }
 
+var (
+	deleteProjectTagScript = `
+		if (ctx._source['projects'] != null) {
+			def foundIndex = ctx._source.projects.indexOf(params.project);
+
+			if (foundIndex != -1) {
+				ctx._source.projects.remove(foundIndex);
+			}
+		}
+	`
+
+	matchingProjectTagsScript = `ArrayList matchingProjects = new ArrayList();
```

Comment:
not really related to the work of this PR but i noticed this script was the exact same in the helper functions that were using it.